src/app/find-help/search-result/search-result.component.ts

```diff
@@ -17,6 +17,8 @@ export class SearchResultComponent implements OnChanges {
   description!: string;
   @Input()
   posted?: Date | string | undefined;
+  @Input()
+  origin?: Location;
```

Comment:
We already have `location` that we treated as `origin` in transport which this change will break.

I guess this is clearer... But we need that update in such a case. I can do it in a followup PR.